Explorer/Assets/DCL/Character/CharacterMotion/Systems/SDKAvatarShapesMotionSystem.cs

```diff
@@ -0,0 +1,150 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using Arch.SystemGroups.Throttling;
+using DCL.AvatarRendering.AvatarShape.UnityInterface;
+using DCL.Character.Components;
+using DCL.CharacterMotion.Animation;
+using DCL.CharacterMotion.Components;
+using DCL.Diagnostics;
+using DCL.Utilities;
+using DCL.Utilities.Extensions;
+using ECS.Abstract;
+using UnityEngine;
+
+namespace DCL.Character.CharacterMotion.Systems
+{
+    [UpdateInGroup(typeof(PresentationSystemGroup))]
+    [LogCategory(ReportCategory.AVATAR)]
+    [ThrottlingEnabled]
+    public partial class SDKAvatarShapesMotionSystem : BaseUnityLoopSystem
+    {
+        private const float DISTANCE_EPSILON = 0.0001f;
+        private const float WALK_DISTANCE = 1.5f;
+        private const float WALK_SPEED = 4f;
+        private const float RUN_SPEED = 10f;
+        private const float ROTATION_SPEED = 6.5f;
+        private const float DOT_THRESHOLD = 0.999f;
+
+        private SDKAvatarShapesMotionSystem(World world) : base(world) { }
+
+        protected override void Update(float t)
+        {
+            UpdateMotionQuery(World, t);
+        }
+
+        [Query]
```

Comment:
Let's add a `[All(typeof(PBAvatarShape))]` to make sure this query ONLY runs for SDK avatars